.here

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this file for?

Reply:
I don't know. Need to ask my colleague Michael Rustler (@mrustl) who is available next week.

Reply:
okay. For now we just keep it here. If we don't need it we do some housekeeping later on